src/hit/api/evaluator/rotations.cpp

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "rotations.h"
+
+#include <glog/logging.h>
+
+#include <iomanip>
+
+#include "../../common.h"
+
+using namespace std;
+
+namespace hit {
+
+    RotationSet::RotationSet(int num_slots) : num_slots_(num_slots) {
+    }
+
+    CKKSCiphertext RotationSet::encrypt(const vector<double> &coeffs) {
+        return encrypt(coeffs, -1);
+    }
+
+    CKKSCiphertext RotationSet::encrypt(const vector<double> &, int level) {
```

Comment:
This appears to ignore the `vector<double>` is that correct?

From greater context, I'm assuming this is correct as we are passing a fake ciphertext through the system to measure how it is handled. Yes?

Reply:
That's basically correct. This evaluator doesn't actually need _any_ inputs, and in a better design (at a fundamental/language level) you could get this information without "encrypting" any inputs at all. However, this is what we have at the moment, so "encryption" just ignores the input, because it doesn't impact the calculation.

Reply:
I was more concerned that it appears to create a meaningless output, but this makes sense.